src/main/java/org/prebid/server/bidder/appnexus/AppnexusBidder.java

```diff
@@ -409,7 +409,7 @@ private BidderBid bidderBid(Bid bid) {
             bid.setCat(Collections.emptyList());
         }
 
-        return BidderBid.of(bid, bidType(appnexus.getBidAdType()), null);
+        return BidderBid.of(bid, bidType(appnexus.getBidAdType()), currency);
```

Comment:
Mb let's check for null and set DEFAULT_CURRENCY if there are no cur from response, to be more clear. (+ last sentence in ticket). 
(I know that we later will be using USD if null)

Reply:
We don't really need to do this since currency service already cares in `org.prebid.server.currency.CurrencyConversionService#convertCurrency`.